### DISCUSSION TITLE
No source file available for this location

### DISCUSSION THREAD

**Discussion by IlyaShaternik** [VerySleepy/verysleepy#101] [Q&A]
I'm sorry if this is stupid question. So I have an executable which was compiled with gcc from MSYS2 MinGW64 with debug symbols. For example if I use gdb on it I can step through the source as expected. I wanted to try to use verysleepy on it. It collects information about how much time is spent in different functions, but the source code is not available. Maybe it is because gcc version is newer than expected by Dr.MinGW?

**Comment by CyberShadow** [comment]
Yes, it's possible.

GCC has an option to configure the DWARF version that the debug information is generated in, perhaps you could try that.

Make sure that Very Sleepy is configured to use the appropriate dbghelp implementation.
